src/parser/parser_visitor.py

```diff
@@ -381,6 +391,37 @@ def visitFromClause(self, ctx: evaql_parser.FromClauseContext):
 
         return {"from": from_table, "where": where_clause}
 
+    def visitOrderByClause(self, ctx: evaql_parser.OrderByClauseContext):
+        # print("visitOrderByClause ---------------")
+        # print("ORDER:", ctx.ORDER())
+        # print("BY:", ctx.BY())
+        # print("orderByExpression:", ctx.orderByExpression())
+        # print("COMMA:", ctx.COMMA())
+
+        orderby_clause_data = []
+        # [(TupleValueExpression #1, ASC), (TVE #2, DESC), ...]
+        for expression in ctx.orderByExpression():
+            orderby_clause_data.append(self.visitOrderByExpression(expression))
+
+        return orderby_clause_data
+
+    def visitOrderByExpression(
+            self, ctx: evaql_parser.OrderByExpressionContext):
+        # print("visitOrderByExpression ---------------")
+        # print("expression:", ctx.expression())
+        # print("ASC:", ctx.ASC())
+        # print("DESC:", ctx.DESC())
+        # print("getRuleIndex:", ctx.getRuleIndex())
+
+        # print(self.visitChildren(ctx.expression()))
+
+        if ctx.DESC():
+            sort_token = evaql_parser.DESC
```

Comment:
We should add a new Enum in the parser.types.py and use that instead of using evaql_parser.{ASC, DESC}